packages/styles/src/mixins/_icons.scss

```diff
@@ -16,32 +17,37 @@
   forced-color-adjust: preserve-parent-color;
 }
 
-@mixin post-icon(
-  $base:
-    'https://unpkg.com/@swisspost/design-system-icons@#{icon-version.$post-icon-version}/public/post-icons',
-  $name,
-  $color: inherit,
-  $width: 1em,
-  $height: 1em
-) {
+$loaded-icons: () !default;
+
+@mixin load-icon-css($name, $path: '../icons/temp') {
+  @if list.index($loaded-icons, $name) == null {
+    $loaded-icons: list.append($loaded-icons, $name) !global;
+    @at-root {
+      @include meta.load-css('#{$path}/#{$name}');
+    }
+  }
+}
+
+@mixin post-icon($name, $color: inherit, $width: 1em, $height: 1em) {
   display: inline-block;
   width: $width;
   height: $height;
   vertical-align: -0.15em;
   background-color: currentColor;
   color: $color;
+  mask-image: var(--post-icon-#{$name});
   /* stylelint-disable-next-line property-no-vendor-prefix */
-  -webkit-mask-image: url('#{$base}/#{$name}.svg');
-  mask-image: url('#{$base}/#{$name}.svg');
+  -webkit-mask-image: var(--post-icon-#{$name});
+  mask-position: center center;
   /* stylelint-disable-next-line property-no-vendor-prefix */
   -webkit-mask-position: center center;
-  mask-position: center center;
+  mask-repeat: no-repeat;
   /* stylelint-disable-next-line property-no-vendor-prefix */
   -webkit-mask-repeat: no-repeat;
-  mask-repeat: no-repeat;
+  mask-size: 100%;
   /* stylelint-disable-next-line property-no-vendor-prefix */
   -webkit-mask-size: 100%;
```

Comment:
Please read the answer, marked as correct: https://stackoverflow.com/questions/7080605/ordering-of-vendor-specific-css-declarations, then make sure for every property the one without vendor prefix comes last.

```scss
.class {
  -webkit-anyprop: value;
  -moz-anyprop: value;
  -ms-anyprop: value;
  anyprop: value;
}
```